src/LinkButton.css

```diff
@@ -0,0 +1,20 @@
+.link-button {
```

Comment:
Interested in ways to avoid duplicating most of this with the button class, which I think I still need because some of the buttons are still just buttons not links.

Reply:
Hmm, but width properties work differently for links and buttons.  If I put the `width: 100%` on both the link and the button, they end up different:

<img width="535" alt="Screen Shot 2020-05-03 at 4 36 18 PM" src="https://user-images.githubusercontent.com/1218629/80925285-748b1100-8d5c-11ea-8f59-4b886612cc32.png">